plugin/src/test/java/org/opensearch/ml/rest/RestCohereInferenceIT.java

```diff
@@ -78,7 +78,9 @@ public void test_cohereInference_withDifferent_postProcessFunction() {
             List output = (List) inferenceResult.get("inference_results");
             assertEquals(errorMsg, 1, output.size());
             assertTrue(errorMsg, output.get(0) instanceof Map);
-            validateOutput(errorMsg, (Map) output.get(0), DATA_TYPE.get(postProcessFunction));
+            if (output != null) {
```

Comment:
Here adding null check for output won't make any difference. If output is null, it would have thrown NullPointerException at line number 79 - output.size().

Here actual issue is in line number 91, outputList.get(0)).get("data_type"). This is becoming null.

Reply:
I don't think we need this PR as the failure IT should already fixed, please check this comment: https://github.com/opensearch-project/ml-commons/pull/3628#issuecomment-2731800996